main/k8s-iam.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_iam_role" "beekeeper_k8s_role" {
```

Comment:
To clarify, is there one container running per pod now? And there are 2 separate deployments? If so, we can be more specific about what policies each pod needs, we can have two different `aws_iam_role` for each pod essentially, one which has secrets/sqs and the other which has secrets/s3

Reply:
Yes - the way I set this up here will do a container per pod w/ 2 separate deployments. And sure thing that's totally possible, let me set that up.

Reply:
updated!